i3pystatus/core/modules.py

```diff
@@ -13,6 +13,7 @@ class Module(SettingsBase):
         ('on_rightclick', "Callback called on right click (see :ref:`callbacks`)"),
         ('on_upscroll', "Callback called on scrolling up (see :ref:`callbacks`)"),
         ('on_downscroll', "Callback called on scrolling down (see :ref:`callbacks`)"),
+        ('hints', "Additional output blocks for module output (see :ref:`hints`)"),
```

Comment:
Yeah sphinx doesn't parse normal strings in the code. It does show correctly in the module reference, though.